src/dataset_impl.h

```diff
@@ -30,7 +30,8 @@
 namespace vsag {
 
 class DatasetImpl : public Dataset {
-    using var = std::variant<int64_t, const float*, const int8_t*, const int64_t*>;
+    using var =
+        std::variant<int64_t, const float*, const int8_t*, const int64_t*, const std::string*>;
```

Comment:
Could you please add the field comments?

Reply:
The explanation of these fields can be found at the interface.